src/pas/plugins/kimug/plugin/__init__.py

```
@@ -199,5 +244,22 @@ def _decode_token(self, token):
+        try:
+            new_user = self._create_user(userid)
+        except Exception:
+            logger.exception("Could not create local user for %s", userid)
+        userinfo = {
+            "username": payload["email"],
+            "email": payload["email"],
+            "name": payload["preferred_username"],
+        }
+        try:
+            self._update_user(new_user, userinfo, first_login=True)
+        except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)